server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosController.java

```diff
@@ -0,0 +1,64 @@
+package com.objectcomputing.checkins.services.kudos;
+
+import com.objectcomputing.checkins.services.role.RoleType;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Delete;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.annotation.Put;
+import io.micronaut.http.annotation.Status;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.List;
+import java.util.UUID;
+
+@Controller("/services/kudos")
+@Secured(SecurityRule.IS_AUTHENTICATED)
+@ExecuteOn(TaskExecutors.BLOCKING)
+@Tag(name = "kudos")
+public class KudosController {
+
+    private final KudosServices kudosServices;
+
+    public KudosController(KudosServices kudosServices) {
+        this.kudosServices = kudosServices;
+    }
+
+    @Post
+    @Status(HttpStatus.CREATED)
+    public Kudos create(@Body @Valid KudosCreateDTO kudos) {
+        return kudosServices.save(kudos);
+    }
+
+    @Put
+    @Secured(RoleType.Constants.ADMIN_ROLE)
```

Comment:
Does this warrant a new permission, or is ADMIN_ROLE sufficient?  I can see why having non-admins who can approve Kudos may be useful

Reply:
This warrants an issue as well. Pulling into develop as is for now.